stack/app/ecs.tf

```diff
@@ -0,0 +1,49 @@
+module "ecs-kms-key" {
+  source = "../kms-key"
+
+  # Required
+  environment = var.environment
+  project     = var.project
+  alias       = "ecs"
+  description = "kms key for ${var.project} ECS"
+
+  # Optional
+  deletion_window_in_days = var.kms_deletion_window_in_days
+}
+
+module "ecs" {
+  source = "../ecs"
+
+  project            = var.project
+  environment        = var.environment
+  vpc_id             = module.vpc.id
+  subnet_private_ids = module.vpc.subnet_private_ids
+  subnet_public_ids  = module.vpc.subnet_public_ids
+  secrets_arns = [
+    module.secrets["app"].arn
```

Comment:
the secrets module now lives in a folder next to this one, i.e. it will live in its own terraform workspace, i.e. we cannot reference it via the `module` function here.